docs/configuration/description.md

```diff
@@ -0,0 +1,67 @@
+Files are placed in the `.mainframer` directory.
+The final configuration is the combination of files placed in:
+
+* `${HOME}/.mainframer` — the global configuration;
+* `.mainframer` — the project configuration.
+
+It is posssible to share the configuration using the global one
```

Comment:
Maybe reword?

"It is possible to declare common configuration using the global one and fine-tune it using the project one"